docs/secrets-and-passwords.md

```diff
@@ -0,0 +1,78 @@
+# Projekt-Secrets und Passwort-Verwaltung
+
+_Eine Dokumentation zur sicheren Verwaltung von Secrets und Passwörtern in Projekten_
+
+---
+
+**Inhaltsverzeichnis**
+
+1. Einleitung
+2. 1Password Vault für Secrets und Passwörter
+3. Verwaltung von Secrets in GitHub
+   - 3.1 Hinzufügen von Secrets in GitHub
+   - 3.2 Verwendung von Secrets in Workflows
+4. Best Practices für Sicherheit
+   - 4.1 Verwendung starker Passwörter
+   - 4.2 Passwortkomplexität
+5. Fazit
+
+---
+
+## 1. Einleitung
+
+Die Sicherheit von Secrets und Passwörtern in einem Projekt ist von entscheidender Bedeutung, um sensible Informationen vor unautorisierten Zugriffen zu schützen. Diese Dokumentation beschreibt bewährte Methoden zur Verwaltung von Secrets und Passwörtern in einem Projektumfeld unter Verwendung von 1Password und GitHub.
+
+## 2. 1Password Vault für Secrets und Passwörter
+
+Alle Secrets und Passwörter eines Projekts müssen in 1Password verwaltet werden. Ein dedizierter Vault für jedes Projekt innerhalb von 1Password ermöglicht eine organisierte und zentrale Ablage der sensiblen Informationen. Zugriff auf diesen Vault sollte auf eine begrenzte Anzahl von vertrauenswürdigen Personen beschränkt sein, ganz getreu dem Motto "So viele wie nötig, so wenige wie möglich". Die Verwendung von 1Password stellt sicher, dass die Daten verschlüsselt und geschützt sind.
```

Comment:
We can also suggest a naming for the vaults (i.e. `client-project` -> `haufe-powerhelferapp`). Gives some guidance for the reader / developer.